### PR TITLE
Print docx Changes

### PR DESCRIPTION
**Pull request by snehar-nd**
Print docx Service for version 1.0 flow
https://project-sunbird.atlassian.net/wiki/spaces/PRD/pages/3033169921/Enabling+Print+Program+Service+for+Refactor+Flow+QuML+1.0

### CODE REVIEW

File: src/envVariables.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const envVariables = {
     SUNBIRD_AUTO_CREATION_TOPIC: process.env.sunbird_auto_creation_topic,
     SUNBIRD_QUESTION_BULKUPLOAD_TOPIC: process.env.sunbird_question_bulkupload_topic,
     SUNBIRD_KAFKA_BULKUPLOAD_CONSUMER_GROUP_ID:process.env.sunbird_kafka_bulkupload_consumer_group_id,
-    SUNBIRD_ASSESSMENT_SERVICE_BASE_URL : process.env.sunbird_assessment_service_base_url,
+    SUNBIRD_ASSESSMENT_SERVICE_BASE_URL : process.env.sunbird_assessment_service_base_url  || 'https://dock.sunbirded.org/action/',
```
Comment (pallakartheekreddy) [inline src/envVariables.js:20]: Please remove this

File: src/routes/print.js [hunk 2]
```diff
@@ -1,16 +1,55 @@
...
 const requestMiddleware = require("../middlewares/request.middleware");
-// const base64 = require('base64topdf');
 const BASE_URL = "/program/v1";
 // Refactor this to move to service
 async function printDocx(req,res){
   const id = req.query.id;
   const format = req.query.format;
-  // buildDOCXwithCallback(function (binary, error, errorMsg) {
+  const version = req.query.version;
+  console.log("Version:",version)
```
Comment (pallakartheekreddy) [inline src/routes/print.js:12]: remove console logs

File: src/routes/print.js [hunk 3]
```diff
@@ -46,50 +86,16 @@ async function printDocx(req,res){
   });
 }
 
-async function printPDF(req, res) {
```
Comment (pallakartheekreddy) [inline src/routes/print.js:49]: why you removed printPDF?

File: src/envVariables.js (hunk 1 above)
Comment (snehar-nd) [reply src/envVariables.js]: removed 'https://dock.sunbirded.org/action/'

File: src/routes/print.js (hunk 2 above)
Comment (snehar-nd) [reply src/routes/print.js]: done

**Review by pallakartheekreddy — APPROVED** [review]

**Pull request merged**